dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/ZenodoHelperTest.java

```diff
@@ -44,4 +47,58 @@ public void extractDoiFromBadDoiUrl() {
         Assert.assertNotEquals("10.5072/zenodo.372767", doi);
     }
 
+    @Test
+    public void checkAliasCreationFromDoiWithInvalidPrefix() {
+        User user = new User();
+        user.setIsAdmin(false);
+        user.setCurator(false);
+        String doi = "drs:10.5072/zenodo.372767";
+        boolean throwsError = false;
+        try {
+            String formattedDoi = ZenodoHelper.createAliasUsingDoi(doi, user);
+        } catch (CustomWebApplicationException ex) {
+            throwsError = true;
+        }
+
+        if (!throwsError) {
+            fail("Non admin or curator was able to create an alias with an invalid prefix.");
+        }
+    }
+
+    @Test
+    public void checkAdminAliasCreationFromDoiWithInvalidPrefix() {
+        User user = new User();
+        user.setIsAdmin(true);
+        user.setCurator(false);
+        String doi = "drs:10.5072/zenodo.372767";
+        boolean throwsError = false;
+        try {
+            String formattedDoi = ZenodoHelper.createAliasUsingDoi(doi, user);
+        } catch (CustomWebApplicationException ex) {
+            throwsError = true;
+        }
+
+        if (throwsError) {
+            fail("Admin was not able to create an alias with an invalid prefix.");
+        }
+    }
+
+    @Test
+    public void checkNonAdminAliasCreationFromValidDoi() {
+        User user = new User();
+        user.setIsAdmin(false);
+        user.setCurator(false);
+        String doi = "10.5072/zenodo.372767";
+        boolean throwsError = false;
+        try {
+            String formattedDoi = ZenodoHelper.createAliasUsingDoi(doi, user);
+        } catch (CustomWebApplicationException ex) {
+            throwsError = true;
```

Comment:
It'd be nice if there was some assertion on the exception in tests

Reply:
Would you please elaborate? Is fail not enough?

Reply:
Many things in the future can cause a CustomWebApplicationException, need to narrow down to exactly what you're expecting the cause of failure to be.

Reply:
Thanks done